src/ray/gcs/gcs_server/gcs_autoscaler_state_manager.h

```diff
@@ -83,6 +83,54 @@ class GcsAutoscalerStateManager : public rpc::autoscaler::AutoscalerStateHandler
     return node_resource_info_;
   }
 
+ protected:
```

Comment:
maybe change the `private` to `protected` is a good choice, although the coding style is not so good.